cookbooks/ros2_windows/recipes/qt5.rb

```diff
@@ -18,52 +18,54 @@
   sensitive true
 end
 
-template  'qt-maintenance.qs' do
-  source 'qt-maintenance.qs.erb'
-  variables Hash[
-    qt_account_email: node['ros2']['qt_account_email'],
-    qt_account_password: node['ros2']['qt_account_password'],
-  ]
-  sensitive true
-end
+#template  'qt-maintenance.qs' do
```

Comment:
Instead of commenting out this code, what do you think of creating two separate qt5.rb files. One for online install and the other for offline?

Reply:
This whole PR is sort of slapdash at present since as the day devolved I focused on getting CI back online.

The original "blame" for this issue and others has been the maintenance script and this PR started as a "quick" comment it out and move on approach which both did not work for infrastructure reasons and because it didn't resolve the underlying issue of download.qt.io being offline.

I do think the correct solution is to create separate recipes for online vs offline installation of Qt and that's likely what will happen with this PR.